install_config/web_console_customization.adoc

```diff
@@ -376,6 +376,41 @@ assetConfig:
 ----
 ====
 
+. As a cluster administrator, run the following command to give users the
```

Comment:
permission to use pipeline strategies should be on by default already, not sure where this is coming from?

i think all strategies are on by default, though we've been discussing turning off custom by default.